airflow/providers/databricks/hooks/databricks.py

```diff
@@ -400,7 +402,13 @@ def get_job_id(self, run_id: str) -> str:
 
     def get_run_state(self, run_id: str) -> RunState:
         """
-        Retrieves run state of the run.
+        Retrieves run state of the run. Please note that any Airflow tasks that call the `get_run_state`
```

Comment:
```suggestion
        """Retrieves run state of the run.

        Please note that any Airflow tasks that call the `get_run_state`
```

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.

Reply:
Also restructuredText requires code to use double ticks, not single.

Reply:
thank you for review, addressed